src/components/ShelterListItem/ShelterListItem.tsx

```diff
@@ -99,6 +99,7 @@ const ShelterListItem = (props: IShelterListItemProps) => {
               ))}
             </div>
           </div>
+          {donationsTags.length > 0 ?  
```

Comment:
Você está usando um ternário que no false ele retorna uma string vazia, nesses casos é melhor usar o operador and (&&):

`{danationsTags.length && (código a ser renderizado)}`